wcivf/apps/people/templates/people/includes/_person_previous_elections_card.html

```diff
@@ -21,10 +21,10 @@ <h3 class="ds-padded">Previous Elections</h3>
         <td>{{ person_post.election.election_date|date:"Y" }}</td>
         <td>{{ person_post.post.label }}: {{ person_post.election }}</td>
         <td>{{ person_post.party.party_name }}</td>
-        {% if person_post.elected %}
-        <td>{{ person_post.votes_cast|intcomma }} votes (elected)</td>
+        {% if person_post.votes_cast %}
+        <td>{{ person_post.votes_cast|intcomma }} votes{% if person_post.elected%} (elected){% endif %}</td>
         {% else %}
-        <td>{{ person_post.votes_cast|intcomma }} votes</td>
+        <td>Vote count not available{% if person_post.elected%} (elected){% endif %}</td>
```

Comment:
Yep, this is the line I was talking about!

I think the logic is slightly different, here in pseudo code:
```
if person_post.elected == True:
    (elected)
elif if person_post.elected == False:
  (not elected)
else:
    Vote count not available

Reply:
Ah makes sense, I'll make that edit, thanks!